### PR TITLE
[5.2] Merge new input into the current request's input array with segmentat…

### PR DESCRIPTION
**Pull request by anteriovieira**
Related to #11405

### CODE REVIEW

File: src/Illuminate/Http/Request.php [hunk 1]
```diff
@@ -517,11 +517,19 @@ protected function retrieveItem($source, $key, $default)
      * Merge new input into the current request's input array.
      *
      * @param  array  $input
-     * @return void
+     * @return $this
```
Comment (GrahamCampbell) [inline src/Illuminate/Http/Request.php]: :-1:

Comment (GrahamCampbell) [inline src/Illuminate/Http/Request.php]: inconstistant with other functions on the request class

Comment (anteriovieira) [inline src/Illuminate/Http/Request.php]: Void then?

Comment (GrahamCampbell) [inline src/Illuminate/Http/Request.php]: yeh, and don't return this in the code

Comment (anteriovieira) [inline src/Illuminate/Http/Request.php]: Done!

**Comment by taylorotwell** [comment]
Holding off on this.

**Pull request closed without merging**